### PR TITLE
Hotfix team place order issues

### PR DESCRIPTION
**Pull request by ian-a-hickey**

### CODE REVIEW

**Comment by gregmoser** [comment]
What is the difference between billing and full?

**Comment by ian-a-hickey** [comment]
Sumit had me add the new context  so I would ask him. He will explain it better anyway.

File: integrationServices/stripe/Payment.cfc [hunk 1]
```diff
@@ -336,20 +336,40 @@ component accessors="true" output="false" displayname="Stripe" implements="Slatw
...
+          httpRequest.addParam(type="formfield", name="card[name]", value="#requestBean.getNameOnCreditCard()#");
+        }
+        if(!isNull(requestBean.getBillingStreetAddress())) {
+          httpRequest.addParam(type="formfield", name="card[address_line1]", value="#requestBean.getBillingStreetAddress()#");
+        }   
+        if(!isNull(requestBean.getBillingStreet2Address())) {
+            httpRequest.addParam(type="formfield", name="card[address_line2]", value="#requestBean.getBillingStreet2Address()#");
+        }
+        if(!isNull(requestBean.getBillingStreet2Address())) {
+          httpRequest.addParam(type="formfield", name="card[address_city]", value="#requestBean.getBillingCity()#");
```
Comment (sumitverma) [inline integrationServices/stripe/Payment.cfc]: Looks like a typo in if, should be city

File: model/service/PublicService.cfc [hunk 2]
```diff
@@ -565,6 +565,28 @@ component extends="HibachiService"  accessors="true" output="false"
 		arguments.rc.$.slatwall.addActionResult( "public:cart.clear", cart.hasErrors() );
 	}
 	
+	/** adds a billing address to an order. uses the billing context. */
```
Comment (sumitverma) [inline model/service/PublicService.cfc]: @ian-a-hickey I think this method and validation is supposed to be in feature and not hotfix, right?

Comment (ian-a-hickey) [inline model/service/PublicService.cfc]: Yes, correct - it was a long day yesterday...I'll make that change now.

**Pull request merged**